geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionDataStore.java

```diff
@@ -3022,7 +3022,7 @@ public void handleInterestEvent(InterestRegistrationEvent event) {
         // If this is a registration event, add interest for this key
         if (isRegister) {
           if (logger.isDebugEnabled()) {
-            logger.debug("PartitionedRegionDataStore for {} adding interest for: ",
+            logger.debug("PartitionedRegionDataStore for {} adding interest for :{}",
```

Comment:
Whitespace change seems wrong.

Reply:
if we search for :{} in the project, we see that it is being used. I remember all logs I analyzed in hydra we used this format. Helps in easier greps